symplyphysics/laws/thermodynamics/isobaric_potential_from_heat_capacity.py

```diff
@@ -0,0 +1,56 @@
+from sympy import (Eq, solve, log)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The isobaric potential of a reaction is a value whose change during a chemical reaction is equal to the change in the internal
+## energy of the system. The isobaric potential shows how much of the total internal energy of the system can be used for chemical
+## transformations.
+
+## Law is: G = H - T * S - Cp * T * (ln(298 / T) + (298 / T) - 1), where
```

Comment:
Do you have a link to this law?

Reply:
I don't have a link to the book to read.
Book "Physico-chemical fundamentals of electronics" Barybin, pages 753-754.
https://biblioclub.ru/index.php?page=book_red&id=75443&razdel=10555

Reply:
Well I cannot verify this formula: we do not have link to formula and we do not have reference numbers.
Maybe you could make a photo of this page?

Reply:
> Well I cannot verify this formula: we do not have link to formula and we do not have reference numbers. Maybe you could make a photo of this page?

![image](https://github.com/blackyblack/symplyphysics/assets/100017508/df8c761e-8ce5-4a93-8b08-4d99ba2adefe)